.github/workflows/building.yml

```diff
@@ -77,6 +80,20 @@ jobs:
         with:
           submodules: 'recursive'
 
+      - name: Free up disk space
```

Comment:
@liruilong940607 for some reason I had to move this step earlier in the pipelines and delete more folder for it to run in github action. We may want to test the piepline in the main branch first and we could do that change in a separate PR to get it working

Reply:
actually seems to work now. Not sure why I have been hitting a problem before